src/Numerics/DGMethods/DGModel.jl

```diff
@@ -87,8 +105,10 @@ function (dg::DGModel)(
     ndrange_interior_surface = Nfp * length(grid.interiorelems)
     ndrange_exterior_surface = Nfp * length(grid.exteriorelems)
 
-    if !increment && num_state_conservative < num_state_tendency
-        tendency .= -zero(FT)
+    if num_state_conservative < num_state_tendency && β != 1
+        # if we don't operate on the full state, then we need to scale here instead of volume_tendency!
+        tendency .*= β
+        β = true
```

Comment:
Why true and not
```suggestion
        β = 1
```

Reply:
`false` is a "strong zero" such that `NaN*false == 0`: the compiler is able to elide it `x*false === 0.0`, which is why i used `Bool`s in the 4-arg form.

`true` doesn't really matter (since `x*1.0 === x`), but we might as well do the same.

Reply:
Makes sense.